### ISSUE TITLE
Make `\Parable\Http\Response` HTTP/2 ready

### ISSUE THREAD

**Issue by FaaPz** [devvoh/parable#16]
My production environment uses HTTP/2, so I suggest something like this to make it future-proof:
```php
header("{$_SERVER['SERVER_PROTOCOL']} {$this->getHttpCode()} {$this->getHttpCodeText()}");
```
http://php.net/manual/en/function.header.php#92305

And please do throw away that array of HTTP status codes. It's not needed, because you can use `http_response_code()`. http://php.net/manual/en/function.http-response-code.php#116880

**Comment by devvoh** [comment]
Will be fixed in 0.12.5.

**Comment by devvoh** [comment]
Should be fixed in 0.12.5. Reopen if it requires fixes!
